Build/.gitignore

```diff
@@ -0,0 +1,412 @@
+
```

Comment:
Why do you need a separate `.gitignore` here?

Reply:
Probably my mistake... I used the file to prevent adding local generated files, but I forgot to remove it. I will remove that.